src/components/AddFarm/AddFarm.js

```diff
@@ -0,0 +1,169 @@
+import {
+  TextField, Button,
+} from '@material-ui/core';
+import React from 'react';
+import './AddFarm.css';
+
+// Airtable set-up here
+const Airtable = require('airtable');
+
+const base = new Airtable({ apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY })
+  .base(process.env.REACT_APP_AIRTABLE_BASE_KEY);
+
+export default class AddFarm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      farmName: null,
+      description: null,
+      /* userID: null, */
+      email: null,
+      phone: null,
+      address: null,
+      zipCode: null,
+      error: null,
+      // showAlert: false,
```

Comment:
Take out comments like showAlert if you're not using it. However, if you want to install material-ui/lab, enter this in your terminal:
npm install @material-ui/lab